Lib/unittest/test/testmock/testmock.py

```diff
@@ -396,6 +396,13 @@ def _check(mock):
         _check(mock)
 
 
+    def test_assert_called_exception_message(self):
+        with self.assertRaisesRegex(AssertionError, "Expected 'mock' to have been called"):
```

Comment:
Nit : Please follow PEP 8 guidelines to ensure the line length is not more than 80 characters here and in below. You can probably have it as a format string with `message` and fill in the mock name like below.

```python
message = "Expected '{0}' to have been called"

with self.assertRaisesRegex(AssertionError, message.format('mock')):
    Mock().assert_called()
```

Reply:
Thanks! What is the preferred way to run pep8 agains the modified code?

Reply:
I only do manual checking. There are some parts of the code that predates PEP8 and PEP8 is enforced only for new code so using pep8 libraries on whole file might give some false information.